MMM-TeslaFi.js

```diff
@@ -322,5 +359,26 @@ Module.register("MMM-TeslaFi", {
     } else {
       return this.numberFormat(valueMiles) + " miles";
     }
+  },
+
+  // Gets Static map as picture
+  getMap: function (lat, lng) {
+    if (this.config.googleMapApiKey !== "") {
+      const options = {
+        center: [lat, lng],
+        zoom: this.config.mapZoom,
+        key: this.config.googleMapApiKey,
+        marker: [lat, lng]
+      };
+      return `https://maps.googleapis.com/maps/api/staticmap?size=${this.config.mapWidth}x${this.config.mapHeight}&center=${options.center}&markers=${options.marker}&key=${options.key}&zoom=${options.zoom}&size=tiny`;
```

Comment:
This gets a little bit messy - could you use [`build-url`](https://www.npmjs.com/package/build-url) instead perhaps?

Reply:
I was trying not to add any more dependencies to the project.

Reply:
No worries - in that case I'm happy to merge this as-is and migrate everything to build-url or similar at a later stage